### ISSUE TITLE
Option to disable VU meter analysis/displays?

### ISSUE THREAD

**Issue by mxmilkiib** [jack-mixer/jack_mixer#102]
I don't need to see the levels moving all the time and it's adding to the system lag I'm getting on this older laptop now I'm running a JACK/PipeWire first system.

Maybe there is a middle ground between bars and nothing that could at least denote there has not been silence on a channel in the last second?

**Comment by SpotlightKid** [comment]
I'm currently working on disabling at least the added processing for the K14 and K20 metering, when it isn't used. I haven't measured whether this actually has a positive performance impact yet.

Adding "no meter" meter scale option should be easy to do. The backend will still update the meter values, but the impact of that is minimal.

**Comment by SpotlightKid** [comment]
Also, note that you can set the meter update interval in `jack_mixer.py`:

https://github.com/jack-mixer/jack_mixer/blob/9c4add82e72b3731d0b3d583c2280105c9d38d3d/jack_mixer.py#L120

It is currently set to 33 (ms). Setting it to 100 or even higher should lessen the burden on your CPU a bit, though the meters are a bit jumpy then.

**Comment by mxmilkiib** [comment]
How about exposing that as a user option?

**Comment by SpotlightKid** [comment]
Yes, that's the plan, but there are a lot of feature requests already, so I can't promise that itmwill happen soon.

**Comment by dsheeler** [comment]
@SpotlightKid, @mxmilkiib: what should be displayed in the "no meter" meter scale case?  Fader takes up whole channel width and meter disappears altogether, or a blank meter that takes up space or ...?

**Comment by dsheeler** [comment]
Would we show the peak values?
